src/chatdMsg.h

```diff
@@ -431,7 +431,11 @@ enum Opcode
       */
     OP_REACTIONSN = 48,
 
-    OP_LAST = OP_HANDLELEAVE,
+    OP_MSGIDTIMESTAMP = 49,
```

Comment:
Missing documentation for those 2 new opcodes. It should indicate the size of params, as usual, and the purpose.

Reply:
Solved it